arbitrator/prover/src/host.rs

```diff
@@ -211,6 +214,7 @@ impl Hostio {
             WavmHaltAndSetFinished => {
                 opcode!(HaltAndSetFinished);
             }
+            MockSchedYield => {}
```

Comment:
Here this function does nothing but as the signature says
> MockSchedYield => func!([], [I32])

it has a return value. So it is better and safer to push an I32 value into the stack.